newIDE/app/src/EventsSheet/ParameterFields/GenericExpressionField/index.js

```diff
@@ -222,12 +222,24 @@ export default class ExpressionField extends React.Component<Props, State> {
     });
   };
 
+  _shouldOpenParametersDialog = (
+    expressionInfo: EnumeratedExpressionMetadata
+  ): boolean => {
+    return expressionInfo.parameters.length > 0;
```

Comment:
Let's add a comment telling that non visible parameters (like "code only" parameters) are already filtered out.